.github/workflows/nightly.yml

```
@@ -85,13 +85,14 @@ jobs:
       
-      - name: Get version info
-        id: version
+      - name: Get build info
+        id: build-info
         shell: bash
         run: |
           echo "version=nightly-$(date +%Y%m%d)" >> $GITHUB_OUTPUT
           echo "commit=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
           echo "date=$(date -u +%Y-%m-%dT%H:%M:%SZ)" >> $GITHUB_OUTPUT
+          echo "trebSolCommit=$(cd treb-sol && git rev-parse HEAD)" >> $GITHUB_OUTPUT
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)